src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.impl;
+
+import db.FruitShopStorage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import service.FruitShopService;
+import service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    private FruitShopService fruitShopService;
+
+    public ReportServiceImpl(FruitShopService fruitShopService) {
+        this.fruitShopService = fruitShopService;
+    }
+
+    @Override
+    public List<String> createReport() {
+        //стоворюємо репорт
```

Comment:
is this comment for those who do not understand what 'createReport' means?))) 
It would not help them